0-bootstrap/main.tf

```diff
@@ -100,14 +100,10 @@ module "seed_bootstrap" {
   ]
 }
 
-module "seed_project_budget" {
-  source               = "terraform-google-modules/project-factory/google//modules/budget"
-  version              = "~> 8.1"
-  billing_account      = var.billing_account
-  projects             = [module.seed_bootstrap.seed_project_id]
-  amount               = var.seed_project_budget_amount
-  alert_spent_percents = var.seed_project_alert_spent_percents
-  alert_pubsub_topic   = var.seed_project_alert_pubsub_topic
+resource "google_billing_account_iam_member" "tf_billing_admin" {
```

Comment:
this lgtm for now
but long term we can investigate something like this
https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/b933b9b883fbf61eb354f9e8679b54c7cb40e560/main.tf#L154